ChattoAdditions/Source/Input/ChatInputBarPresenter.swift

```diff
@@ -109,17 +109,13 @@ public class BasicChatInputBarPresenter: NSObject, ChatInputBarPresenter {
     // MARK: Input View
 
     private weak var currentInputView: InputContainerView?
-
+    
     private func updateHeight(for inputView: InputContainerView) {
         inputView.contentHeight = {
-            if let keyboardHeight = self.lastKnownKeyboardHeight, keyboardHeight > 0 {
+            if let keyboardHeight = self.lastKnownKeyboardHeight, keyboardHeight > UIScreen.minimumKeyboardHeight {
                 return keyboardHeight
             } else {
-                if UIApplication.shared.statusBarOrientation.isPortrait {
-                    return UIScreen.main.defaultPortraitKeyboardHeight
-                } else {
-                    return UIScreen.main.defaultLandscapeKeyboardHeight
-                }
+                return UIScreen.minimumKeyboardHeight
```

Comment:
I believe, that this implementation will set a minimum keyboard height all the time for a custom input view if it was opened before a keyboard. It shouldn't work like this. In this case keyboard height should depende on a current device and orientation.

Minimum keyboard height should be used only when we definately know a keyboard height and it's less then minimum keyboard height. I would suggest to test everything with this change:

```swift
if let keyboardHeight = self.lastKnownKeyboardHeight, keyboardHeight > 0 {
    return max(keyboardHeight, UIScreen.minimumKeyboardHeight)
} else {
    // old implementation
}
```